app/code/Magento/User/Controller/Adminhtml/Auth.php

```diff
@@ -18,19 +19,28 @@
      * @var \Magento\User\Model\UserFactory
      */
     protected $_userFactory;
+    
+    /**
+     * @var \Magento\Backend\Helper\Data
+     */
+    protected $_backendHelper;
 
     /**
      * Construct
      *
      * @param \Magento\Backend\App\Action\Context $context
      * @param \Magento\User\Model\UserFactory $userFactory
+     * @param \Magento\Backend\Helper\Data $backendHelper
      */
     public function __construct(
         \Magento\Backend\App\Action\Context $context,
-        \Magento\User\Model\UserFactory $userFactory
+        \Magento\User\Model\UserFactory $userFactory,
+        \Magento\Backend\Helper\Data $backendHelper = null
```

Comment:
Maybe you could do the PR on `2.3-develop` instead for the refactoring. Than you can remove the `= null` part and you don't need the if statement on line 42 and 43.

Reply:
I can do a forwardport for 2.3-develop.